src/main/java/com/appland/appmap/output/v1/CodeObject.java

```diff
@@ -194,11 +195,24 @@ public CodeObject(CodeObject src) {
    * @return An estimated source file path
    */
   public static String getSourceFilePath(CtClass classType) {
-    String[] parts = {
-      "src", "main", "java",
-      classType.getPackageName().replace('.', '/'),
-      classType.getClassFile().getSourceFile()
-    };
+    final List<String> parts = new ArrayList(Arrays.asList("src", "main", "java"));
```

Comment:
I thought we weren't doing this. In VSCode, if the project structure doesn't match this convention, it can actually break the "View source" link.

Reply:
We don't have a better alternative right now. The source path we can grab from the jar is partial and may match many paths. We have another branch open to explore better solutions.

Reply:
Just to be clear, this heuristic is more likely to resolve to a real file than to not include it until we have a better solution in place.

Reply:
No, I think we should pull this out. It makes things worse in VSCode which is the #1 target environment.